crates/tabby-common/src/api/event.rs

```diff
@@ -49,6 +51,11 @@ pub enum Event {
         #[serde(skip_serializing_if = "Option::is_none")]
         user: Option<String>,
     },
+    ChatCompletion {
+        completion_id: String,
+        prompt: Arc<String>,
```

Comment:
log input prompt as format of `Vec<Message>`, and output in format of `Message`

https://github.com/TabbyML/tabby/blob/0977b78e6555a3faa7ec825046f461c15fd9315d/crates/tabby/src/services/chat.rs#L26

Reply:
Can't use `Vec<Message>` in the event because `Event` is in `tabby-common` and `Message` is in `tabby`.

Reply:
You could duplicate the `Message` here, just like what we did for `Segments` (which is also a duplication of what defined in `tabby`)

This is also a best practice when storing serialized json - to separate the schema definition between what filled in request / what got logged.